src/ert/gui/tools/manage_cases/case_init_configuration.py

```diff
@@ -262,3 +261,8 @@ def _showInfoForCase(self, case_name=None):
         html += "</table>"
 
         self._case_info_area.setHtml(html)
+
+    @showWaitCursorWhileWaiting
+    def on_tab_changed(self, p_int):
+        if p_int == 3:
```

Comment:
This is probably fine, but can we instead do something like:
``` python
if case_widget.currentWidget().objectName() is "html_text":
    self._showInfoForCase()
```
I dislike the fixed indices, because re-ordering will break stuff.
But that being said, if there is no super-easy alternative to this, we should let this pass.

Reply:
We can do something like:

```python 
if self.tabText(p_int) == "Case info":
    self._showInfoForCase()
```

In this case, the renaming will break stuff, but maybe magic strings are better than magic numbers.

Reply:
Sure. This would be equivalent to my proposal.
I see i had 'case_widget' where i should have had 'self'
Both rely on the object-text or object-name.